lib/intelligent_foods/resources/order.rb

```diff
@@ -48,7 +48,7 @@ def cancel!
     def request_body
       @request_body ||= {
         menu_id: menu.id,
-        reference_id: external_id,
+        reference_id: external_id.to_s,
```

Comment:
should we not have a test to ensure that this doesn't regress? if we're enforcing to_s to ensure there is no type errors